### PR TITLE
README.md: typo

### PR DESCRIPTION
**Pull request by flyskywhy**
require is @textileio not @textile in ## Usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,12 +171,12 @@ Textile's JS HTTP Client is a work in progress. As such, there's a few things yo
   * Take a look at [textile-go](https://github.com/textileio/textile-go) (which we intend to follow to a point), and also at some of the client repositories: for instance, [`textile-mobile`](https://github.com/textileio/textile-mobile) and the Textile [`react-native-sdk`](https://github.com/textileio/react-native-sdk). Contributions here that would be most helpful are **top-level comments** about how it should look based on our understanding. Again, the more eyes the better.
   * **Add tests**. There can never be enough tests.
   * **Contribute to the [Textile WIKI](https://github.com/textileio/textile-go/wiki)** with any additions or questions you have about Textile and its various impmenentations. A good example would be asking, "What is a thread?". If you don't know a term, odds are someone else doesn't either. Eventually, we should have a good understanding of where we need to improve communications and teaching together to make Textile even better.
-  
+
  Before you get started, be sure to read our [contributors guide](./CONTRIBUTING.md) and our [contributor covenant code of conduct](./CODE_OF_CONDUCT.md).
 
 ### Contributors
 
-Carson Farmer <carson@textile.io>
```
Comment (carsonfarmer) [inline README.md:179]: Looks like your linter removed trailing spaces. While this is probably really nice in general, here we want the contributors to show as a 'list'. Maybe instead, turn this into a `ul` using `*` in front of each contributor?

**Review by carsonfarmer — COMMENTED** [review]
This looks great, thanks for submitting our first external PR 🥇! One tiny 'nit' and then should be good to go!

File: README.md (hunk 1 above)
Comment (flyskywhy) [reply README.md:179]: OK, `*` is used in the bellow commit :stuck_out_tongue_closed_eyes:  , for many text editor will automatically remove trailing spaces by default.

**Review by carsonfarmer — APPROVED** [review]
Awesome, thanks!

**Pull request merged**